CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+
+Contributing to pyDICOM
+============================
+
+This is summary for contributing code, documentation, testing, and filing
+issues.Please read it carefully to help make the code review process go as
+smoothly as possible and maximize the likelihood of your contribution being
+merged.
+
+How to contribute
+-----------------
+
+The preferred workflow for contributing to pydicom is to fork the
+[main repository](https://github.com/pydicom/pydicom) on
+GitHub, clone, and develop on a branch. Steps:
+
+1. Fork the [project repository](https://github.com/pydicom/pydicom)
+   by clicking on the 'Fork' button near the top right of the page. This creates
+   a copy of the code under your GitHub user account. For more details on
+   how to fork a repository see [this guide](https://help.github.com/articles/fork-a-repo/).
+
+2. Clone your fork of the pydicom repo from your GitHub account to your local disk:
+
+   ```bash
+   $ git clone git@github.com:YourLogin/pydicom.git
+   $ cd pydicom
+   ```
+
+3. Create a ``feature`` branch to hold your development changes:
+
+   ```bash
+   $ git checkout -b my-feature
+   ```
+
+   Always use a ``feature`` branch. It's good practice to never work on the ``master`` branch!
+
+4. Develop the feature on your feature branch. Add changed files using ``git add`` and then ``git commit`` files:
+
+   ```bash
+   $ git add modified_files
+   $ git commit
+   ```
+
+   to record your changes in Git, then push the changes to your GitHub account with:
+
+   ```bash
+   $ git push -u origin my-feature
+   ```
+
+5. Follow [these instructions](https://help.github.com/articles/creating-a-pull-request-from-a-fork)
+to create a pull request from your fork. This will send an email to the committers.
+
+(If any of the above seems like magic to you, please look up the
+[Git documentation](https://git-scm.com/documentation) on the web, or ask a friend or another contributor for help.)
+
+Pull Request Checklist
+----------------------
+
+We recommended that your contribution complies with the
+following rules before you submit a pull request:
+
+-  Follow the
+   [coding-guidelines](http://pydicom.org/dev/developers/contributing.html#coding-guidelines).
+
+-  Use, when applicable, the validation tools and scripts in the
+   `sklearn.utils` submodule.  A list of utility routines available
+   for developers can be found in the
+   [Utilities for Developers](http://pydicom.org/dev/developers/utilities.html#developers-utils)
+   page.
+
```

Comment:
In that case, when they do the PR our CI would scream at them. I think that when you require to activate your own CI, you already know what you are doing. Otherwise, you are putting lots of pressure to newcomers.